source/Calamari.Tests/AWS/CloudFormation/CloudFormationFixtureHelpers.cs

```diff
@@ -0,0 +1,159 @@
+﻿using System;
+using System.IO;
+using System.Threading.Tasks;
+using Amazon.CloudFormation;
+using Amazon.Runtime;
+using Amazon;
+using Amazon.S3;
+using Calamari.Aws.Commands;
+using Calamari.Aws.Deployment;
+using Calamari.Aws.Integration.CloudFormation;
+using Calamari.Common.Features.Packages;
+using Calamari.Common.Features.Processes;
+using Calamari.Common.Plumbing.FileSystem;
+using Calamari.Common.Plumbing.Logging;
+using Calamari.Common.Plumbing.Variables;
+using Calamari.Tests.Helpers;
+using FluentAssertions;
+
+namespace Calamari.Tests.AWS.CloudFormation
+{
+    public static class CloudFormationFixtureHelpers
+    {
+        
+        public static string GetBasicS3Template(string stackName)
+        {
+            return $@"
+    {{
+        ""Resources"": {{
+            ""{stackName}"": {{
+                ""Type"": ""AWS::S3::Bucket"",
+                ""Properties"": {{ 
+                    ""BucketName"": {stackName}
+                }}
+            }}
+        }}
+    }}";
+        }
+
+        public static string WriteTemplateFile(string template)
+        {
+            var templateFilePath = Path.GetTempFileName();
+            File.WriteAllText(templateFilePath, template);
+            return templateFilePath;
+        }
+
+        public static async Task ValidateS3BucketExists(string stackName)
+        {
+            await ValidateS3(async client =>
+            {
+                // Bucket with replaced name was created successfully
```

Comment:
nit: The comment here looks like its out of date, its mentioning a replaced name